packages/commonwealth/server/routes/createAddress.ts

```diff
@@ -1,24 +1,22 @@
 import type { WalletId, WalletSsoSource } from '@hicommonwealth/core';
 import type { AddressAttributes, DB } from '@hicommonwealth/model';
-import type { NextFunction } from 'express';
 import type { TypedRequestBody, TypedResponse } from '../types';
 import { success } from '../types';
 import { createAddressHelper } from '../util/createAddressHelper';
 
 export const Errors = {
   NeedAddress: 'Must provide address',
-  NeedChain: 'Must provide chain',
+  NeedCommunity: 'Must provide community',
```

Comment:
getting this error when trying to sign in to /ux with keplr. The payload still has `chain` instead of `community`

Reply:
Fixed

Reply:
Confirmed.